libs/libc/obstack/lib_obstack_vprintf.c

```diff
@@ -0,0 +1,77 @@
+/****************************************************************************
+ * libs/libc/obstack/lib_obstack_vprintf.c
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <obstack.h>
+#include <stdio.h>
+#include <assert.h>
+#include <nuttx/streams.h>
+
+/****************************************************************************
+ * Private Types
+ ****************************************************************************/
+
+struct outstream
+{
+  struct lib_outstream_s public;
+  FAR struct obstack *h;
+};
+
+/****************************************************************************
+ * Private Functions
+ ****************************************************************************/
+
+static int lib_puts(FAR struct lib_outstream_s *this,
+    FAR const void *buf, int len)
+{
+  FAR struct outstream *mthis = (FAR struct outstream *)this;
+
+  DEBUGASSERT(this);
+
+  obstack_grow(mthis->h, buf, len);
+
+  return len;
+}
+
+static void lib_putc(FAR struct lib_outstream_s *this, int ch)
```

Comment:
```suggestion
static void obstack_putc(FAR struct lib_outstream_s *this, int ch)
```

Reply:
Ok

Reply:
Not change

Reply:
Oh..  I see. It was in hidden comments for me because I responded to it. The changes got stuck in the stash for these two files. My fault.